service_invocation/csharp/http/config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: config
+spec:
+  features:
+    - name: Resiliency
```

Comment:
This whole need for configuration goes away in a future release right?

Reply:
@paulyuk yes, as Resiliency is targeting Stable in 1.10, this configuration will no longer be relevant.